example/android/build.gradle

```diff
@@ -3,9 +3,9 @@
 buildscript {
     ext {
         buildToolsVersion = "31.0.0"
-        minSdkVersion = 21
-        compileSdkVersion = 31
-        targetSdkVersion = 31
+        minSdkVersion = rootProject.ext.minSdkVersion
+        compileSdkVersion = rootProject.ext.targetSdkVersion
+        targetSdkVersion = rootProject.ext.targetSdkVersion
```

Comment:
👍

Reply:
Unfortunately it seems example android project can't access rootProject's data.